classes/submission/maps/Schema.php

```diff
@@ -817,6 +837,14 @@ protected function getPropertyStages(Enumerable $stageAssignments, Enumerable $r
                 'editorAssigned' => false,
                 'currentUserAssignedRoles' => [],
             ];
+
+            if ($stageId === WORKFLOW_STAGE_ID_EDITING) {
+                $stages[$stageId]['uploadedFilesCount'] = $stageFiles->filter(fn(SubmissionFile $file) => $file->getData('fileStage') == SubmissionFile::SUBMISSION_FILE_COPYEDIT)->count();
+            } else {
+                // A `null` value is used to indicate that no count data is available.
+                // This is also done to ensure that all stage objects has the same properties
+                $stages[$stageId]['uploadedFilesCount'] = null;
```

Comment:
An alternative approach I considered was to include the count as a top level property on the submission object.
Something like `copyeditStageFilesCount`.  Thus avoiding a property with mix value on the stage objects.

Reply:
As discussed today, I think having this be nullable here rather than at the top level is the way to go for this. 👍